OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -21,9 +21,34 @@
 
 namespace OpenRA.Mods.Common.Traits
 {
+	[Flags]
+	public enum FlightDynamic
+	{
+		None = 0,
+		MoveIntoShroud = 1,
+		Slide = 2,
+		Hover = 4,
+		VTOL = 8,
+		VTOLTurnToLand = 16,
```

Comment:
#16509 makes this relevant for non-VTOL craft as well, so I'd prefer if the name change was reverted.